apps/wallet-ui/src/components/ui/toast.tsx

```diff
@@ -0,0 +1,129 @@
+"use client";
```

Comment:
the [sonner](https://ui.shadcn.com/docs/components/sonner) stuff feels superior to this imo